ports/nrf/Makefile

```diff
@@ -111,18 +113,37 @@ SRC_C += \
 	drivers/bluetooth/ble_uart.c \
 	boards/$(BOARD)/board.c \
 	nrfx/mdk/system_$(MCU_SUB_VARIANT).c \
+	nrfx/hal/nrf_nvmc.c \
 	device/$(MCU_VARIANT)/startup_$(MCU_SUB_VARIANT).c \
+	background.c \
```

Comment:
Please place this above (around line 107) with the other files in the nrf directory.